pkg/sql/plan/partition.go

```diff
@@ -73,6 +73,7 @@ func buildHashPartition(partitionBinder *PartitionBinder, partitionOp *tree.Part
 		return err
 	}
 
+	partitionInfo.PartitionMsg = tree.String(partitionOp, dialect.MYSQL)
```

Comment:
use tree.String is not a good idea。
some special string like ',  "   will removed in tree.String
you can not get origin PartitionOp from this string.

Reply:
There is no good processing strategy for this, and the string () method of the abstract syntax tree can be reconstructed later